tfe/resource_tfe_workspace_migrate.go

```diff
@@ -95,6 +95,12 @@ func resourceTfeWorkspaceResourceV0() *schema.Resource {
 							Type:     schema.TypeString,
 							Required: true,
 						},
+
+						"tags_regex": {
```

Comment:
learning: What do we actually migrate here? I would argue that v0 of workspace didn't have `tags_regex`. I might be wrong about the purpose of migrations in the tfe provider.

Reply:
Yeah I don't think we need this. This is called from [here](https://github.com/hashicorp/terraform-provider-tfe/blob/d55c76c7e4d839b5712a125adcec84c5a8557a9e/tfe/resource_tfe_workspace.go#L28) and its [about state migration](https://www.terraform.io/plugin/sdkv2/resources/state-migration). We don't have tags_regex for in state, so no need to migrate anything.